docs/source/model_design/designing_vivarium_model/interactive_sim/index.rst

```diff
@@ -296,8 +296,43 @@ choose to make plots in the interactive sim instead!
 Common Challenges
 -----------------
 
-.. todo::
+Using the interactive sim is fundamentally different than looking at simulation outputs 
+and this can lead to challenges. Because you are running a mini simulation on your computer, 
+you overlap a lot more with engineering workflow and have to watch out for some common pitfalls.
+
+1. Using the correct branch and simulation version: 
+
+Be sure that you are pulling from the engineering repo as needed. If you forget, the latest work 
+might not be present in your version of the sim, leading to confusion. 
+
+If you need to look at an old version of the sim or are actively debugging something, you might 
+not work from the main branch. Work with engineering to ensure you're on the correct branch for 
+your needed task. 
+
+This can be especially challenging if you need to use different branchs of upstream repos, like 
+vivarium or vivarium public health. If you run into this type of situation, consult with the 
+engineers to find the best strategy to move forward. 
+
+2. Editing engineering files on your local machine: 
+
+In some cases, you might make edits to the model files on your local machine. This might be to 
+remove certain observers, or model components. While small changes like population or 
+scenario can be done in a notebook, removing components cannot. 
+
+Be careful!! While it's fine to make edits you need, don't push them to the engineering repo. 
+Also, track the edits so if you have to pull a new model version from engineering, you can reset 
+the interactive sim to how you need. 
+
+If you are making edits on your local machine, be sure to write down what changes you make in the 
+notebook so you can refer back to it later. Also consider using :code:`git stash` and 
+:code:`git stash apply` to save and reapply changes. 
+
+3. Differing environments: 
 
-  Add information on: environment management, editable installs of packages within environments 
-  If you remove observers or change things it can have weird effects - talk with engineering 
+Engineering generally uses a different environment to you, which means it can use different 
+versions of packages or of Python. This can cause confusion if something isn't running as 
+expected. 
 
+Usually we use editable installs which will automatically update environments to what is in 
```

Comment:
Editable installs actually don't help with this. I think we can remove this paragraph, but I'll suggest an extra debugging step we can include in the paragraph above.

Reply:
@zmbc Do you think Syl should cut this paragraph, or is there a way to revise it that you can suggest?

Reply:
I suggested removing it, because we actually _don't_ have any good solution to ensure a research environment is the same as an engineering environment. I suggested adding a sentence above, which suggests to re-create the environment, but this is more of a heuristic that will sometimes help than a solid solution.